source/game/field/obj/ObjectBase.hh

```diff
@@ -41,6 +41,16 @@ public:
 
     [[nodiscard]] virtual const char *getName() const;
 
+    /// @addr{0x80821DEC}
+    virtual void disableCollision() {
+        m_boxColUnit->m_flag.setBit(Field::eBoxColFlag::Intangible);
+    }
+
+    // @addr{0x808D6F60}
+    virtual void enableCollision() {
+        m_boxColUnit->m_flag.resetBit(Field::eBoxColFlag::Intangible);
+    }
+
```

Comment:
Since you gutted more stuff, we no longer need `disableCollision()` for the time being. See my below comment regarding `enableCollision()`